cps.py

```diff
@@ -111,6 +111,9 @@ def __init__(self, json_data, prefix=None, package=None):
         self.kind = _get('type', json_data, required=True)
         self.configurations = _make_dict(Configuration, 'configurations',
                                          json_data, prefix, package, self)
+        # Supplemental schema
+        supplemental_schema = ['description', 'license']
```

Comment:
#7 was well and good, but this is an excellent candidate for something that *should* be a class variable :smile:.